src/mos.c

```diff
@@ -276,6 +277,35 @@ char * mos_strtok_r(char *s1, const char *s2, char **ptr) {
 	return s1;
 }
 
+//Alternative to missing strnlen() in ZDS libraries
+size_t mos_strnlen(const char *s, size_t maxlen) {
+    size_t len = 0;

```

Comment:
I suspect your editor isn't set up to match the general style for C code in MOS...

these files generally use tabs rather than spaces.  for the most part, I believe a tab size of 4 works for the C files; the assembler files seem to be written assuming a tab size of 8